foundations/powerPlatform/readme.md

```diff
@@ -203,6 +203,9 @@ The following section describes the design considerations and the design recomme
 - Environments can be created with - or without Dynamics 365 application templates available in the tenant.
 - Dynamics 365 applications takes a dependency on environments with Dataverse provisioned.
 - Dataverse for Microsoft Teams can be created directly from the Teams client, which grants the creator Owner permission on the Environment.
+- Managed Environments requires that the users are licensed, as all apps and flows in a managed environment are premium.
```

Comment:
Reword: Enabling Managed Environments promotes all recourses within an environment to premium. Users who access premium resources must be licensed with a premium license: https://docs.microsoft.com/en-us/power-platform/admin/signup-for-powerapps-admin